crates/frontend/src/utils.rs

```diff
@@ -459,14 +459,29 @@ where
         .map_err(|err| err.to_string())?;
 
     let status = response.status();
-    if status.is_client_error() || status.is_server_error() {
+    if status.is_client_error() {
```

Comment:
Why make this change?

Reply:
https://github.com/juspay/superposition/pull/213#pullrequestreview-2265521287